src/ScaleUnitManagement/UserConfig.sample.xml

```diff
@@ -3,26 +3,15 @@
   
   <!-- Use this flag to indicate whether you want to use one single one-box to host both the hub and the scale unit -->
   <UseSingleOneBox>false</UseSingleOneBox>
+  <!-- If empty, the existing value in web.config will be used - otherwise the web.config will get overwritten with this value. -->
```

Comment:
I'm wondering if this should be part of the AuthConfiguration of each scale unit instance